### DISCUSSION TITLE
Accordion is not having smooth effect.

### DISCUSSION THREAD

**Discussion by djayMehrotra** [microsoft/fluentui#24443] [General] [answered]
There is no smooth effect on accordion open/close (collapsible) action. And if I try using 
`...shorthands.transition('opacity', '0.5s', 'ease-out')` its not applying. Please help **what and how** to do.

**Comment by tomi-msft** [comment, marked as answer]
You can use `makeStyles` to apply the transition effect directly to the `AccordionHeader`. Take a look at this codesandbox: https://codesandbox.io/s/dazzling-feynman-fmtu5w?file=/example.tsx

**Reply by tombohub** [reply]
It doesn't move smooth for me. It's very abrupt. Does it move smooth for you?
